chair.md

```diff
@@ -0,0 +1,5 @@
+# PyCon Conference Chair
+
+The conference chair is the main person in charge. He�s responsible for setting the budget, monitoring expenses and general oversight of all other committees. He works together with the Python.PH Board of Trustees for behind-the-scenes decision making, and to ensure that the conference will be a success.
```

Comment:
Do you think it is worthwhile to make this gender-neutral? I can offer suggestions (read: commit on this branch for everyone else's review) on how to rewrite this such that it avoids gender-specific nouns as well as the awkward "he/she" usage.